### DISCUSSION TITLE
Continue pictures analysis, where I left

### DISCUSSION THREAD

**Discussion by igiuseppem** [sepinf-inc/IPED#2010] [Q&A] [answered]
Hello.
I have a problem or maybe not. Recently I acquired devices that had huge amount of pictures and videos. In this scenario, it's usual that I can't finish the analysis of pictures in few hours or days, so it would be useful that when I close IPED, I could start exactly from the last picture I viewed and continue the analysis of pictures. I have tried to filter pictures selecting only "checked" files, then I have clicked the last selected picture and I have removed the filter to view all files, but the gallery goes at top again. I tried to search the name of the last files I selected, but altough if I find it and I select it, when I cancel the name in the search bar, the gallery goes at top again.
What's wrong?

**Comment by lfcnassif** [comment, marked as answer]
Hi @igiuseppem, there is nothing wrong, the tool simply doesn't have such feature of keeping the previous selection/highlighted item when the current filter or search criteria changes. Possibly the previous selected item won't be in the new results, but if it is, we can try to implement that behavior.

For now, I would suggest you to create a temporary bookmark (or use the item checkbox if not being used yet) to track the last viewed/analyzed image or video. When you open the analysis UI again, apply your desired filters, sort by the bookmark column (or the checkbox one), scroll to the created temporary bookmark (or checkbox), select the item, and then apply your desired sorting criteria/column. It should position you in the point where your analysis stopped. Just be aware the tool remembers the last 2 sorted columns: the last (primary) column and the previous (secondary) one, used as a secondary sorting criteria when the values of the first column are equal.

**Comment by igiuseppem** [comment]
Hi @lfcnassif.
Thank you for your answer. I have applied your suggestion and it works, but I need one more information.
Which is the predefined order for the pictures (or videos) in the table view or in the gallery? Let's say that, after I sort the checkbox column, I am not able to restore the predefined order the pictures (or videos) are displayed. If I estabilish, before to start the new analysis, to sort the pictures by name (for example), obviously I analyze the picture in that order, then to start again I will use that criteria, but I have started in the past days an analysis where I haven't changed the predefined order and after I individuated the last picture checked, I don't know how to go back to the previous sort.

**Reply by lfcnassif** [reply]
Hi @igiuseppem. The default ("predefined") order is the items processing order. Unfortunately there isn't a simple button to return back to it or to disable sorting. Since the application tracks the last 2 sorted columns, you may return back to the default order by sorting by 2 empty columns in sequence or 2 columns where all items have the same value, for example, the Score column (if you aren't doing a term search), the Bookmarks column (if it is empty, i.e. if you haven't created a bookmark yet) or the checkbox column if checkboxes aren't being used (or you can save checked items in a new bookmark to clear them).

**Reply by igiuseppem** [reply]
Thank you @lfcnassif. Your suggestions has been useful.
